data/json/mapgen/bike_shop.json

```diff
@@ -0,0 +1,159 @@
+[
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "s_bike_shop" ],
+    "object": {
+      "fill_ter": "t_grass",
+      "rows": [
+        "aaaaaaaaaaaaaaaaaaaaaaaa",
+        "aaaaaaaaaaaaaaaaaaaaaaaa",
+        "   aaaaaaaaaaaaaaaaaa   ",
+        "   aaaaaaaaaaaaaaaaaa   ",
+        "   aaaaaaaaaaaaaaaaaa   ",
+        "   aaaaaaaaaaaaaaaaaa   ",
+        "   bbbbbbbbeeebddbeeb   ",
+        "   bcccccfbcccccccccb   ",
+        "   bcccccfbcccccccccb   ",
+        "   bcccccfbcccccccccb   ",
+        "   bcccccfbcccccccccb   ",
+        "   bccccccbbccccccccb   ",
+        "   bccccccfbccccccccb   ",
+        "   bccccccfbccfccccgb   ",
+        "   bcccccccccchccccgb   ",
+        "   bccccccccccfccccgb   ",
+        "   bccccccccccfccccgb   ",
+        "   bbbbbbbbbccfcccccb   ",
+        "           bccfcgggcb   ",
+        "           bbbbbbbbbb   ",
+        "      k                 ",
+        "                        ",
+        "                        ",
+        "                        "
+      ],
+      "terrain": {
+        " ": [ "t_dirt", "t_grass", "t_grass", "t_grass" ],
+        "a": "t_sidewalk",
+        "b": "t_wall",
+        "c": "t_floor",
+        "d": "t_door_locked",
+        "e": "t_wall_glass",
+        "f": "t_floor",
+        "g": "t_floor",
+        "h": "t_console_broken",
+        "i": "t_floor",
+        "j": "t_door_c",
+        "k": "t_tree"
+      },
+      "furniture": { "f": "f_counter", "g": "f_rack" },
+      "place_loot": [
+        { "group": "bikeshop_tools", "chance": 80, "repeat": [ 1, 2 ], "x": [ 9, 9 ], "y": [ 7, 10 ] },
+        { "group": "bikeshop_tools", "chance": 80, "repeat": [ 1, 2 ], "x": [ 10, 10 ], "y": [ 12, 13 ] },
+        { "group": "bikeshop_for_sale", "chance": 100, "repeat": [ 1, 2 ], "x": [ 16, 18 ], "y": 18 },
+        { "group": "bikeshop_for_sale", "chance": 100, "repeat": [ 1, 3 ], "x": 19, "y": [ 13, 16 ] }
+      ],
+      "place_vehicles": [
+        { "vehicle": "bikeshop", "x": [ 4, 6 ], "y": [ 3, 4 ], "chance": 90, "rotation": 270 },
+        { "vehicle": "bikeshop", "x": [ 7, 10 ], "y": 4, "chance": 90, "rotation": 270 },
+        { "vehicle": "bikeshop", "x": [ 11, 14 ], "y": 4, "chance": 90, "rotation": 270 },
+        { "vehicle": "bikeshop", "x": 12, "y": 8, "chance": 100, "rotation": 270, "status": 0 },
+        { "vehicle": "bikeshop", "x": 19, "y": 8, "chance": 100, "rotation": 270, "status": 0 },
+        { "vehicle": "bikeshop", "x": [ 5, 6 ], "y": [ 8, 9 ], "chance": 50, "rotation": 0 },
+        { "vehicle": "bikeshop", "x": [ 5, 6 ], "y": [ 10, 11 ], "chance": 50, "rotation": 0 },
+        { "vehicle": "bikeshop", "x": [ 5, 7 ], "y": [ 12, 13 ], "chance": 50, "rotation": 0 },
+        { "vehicle": "bikeshop", "x": [ 5, 7 ], "y": [ 14, 15 ], "chance": 50, "rotation": 0 }
```

Comment:
Don't specify 0 rotation. It is the default.

Reply:
Done.